consensus/src/twins_test.rs

```diff
@@ -162,24 +183,42 @@ impl SMRNode {
             config.base.waypoint = WaypointConfig::FromConfig(waypoint);
             config.consensus.proposer_type = proposer_type;
             // Use in memory storage for testing
```

Comment:
Is this still relevant btw?

Reply:
Not sure, I inherited the Waypoint line from previous chained_bft_smr code. If it's not relevant any more, we can remove it.